src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,205 @@
+        }
+        createRelation(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String getCarById = "SELECT cars.id, cars.model, cars.manufacturer_id, manufacturers.name,"
+                + " manufacturers.country "
+                + "FROM taxi_service.cars JOIN taxi_service.manufacturers "
+                + "ON cars.manufacturer_id = manufacturers.id "
+                + "WHERE cars.id = ? AND cars.is_deleted = FALSE;";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getCarStatement = connection.prepareStatement(getCarById)) {
+            getCarStatement.setLong(1, id);
+            ResultSet resultSet = getCarStatement.executeQuery();
+            if (resultSet.next()) {
+                car = createCarByResultSet(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get car by id = " + id, e);
+        }
+        if (car != null) {
+            car.setDrivers(getDriversByCarId(id));
+        }
+        return Optional.ofNullable(car);
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String getAllCars = "SELECT cars.id, cars.model, cars.manufacturer_id,"
+                + " manufacturers.name, manufacturers.country "
+                + "FROM cars JOIN manufacturers ON cars.manufacturer_id = manufacturers.id "
+                + "WHERE cars.is_deleted = FALSE;";
+        List<Car> carList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllCarsStatement = connection.prepareStatement(getAllCars)) {
+            ResultSet resultSet = getAllCarsStatement.executeQuery();
+            while (resultSet.next()) {
+                carList.add(createCarByResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t get all car from DB", e);
+        }
+        carList.forEach(c -> c.setDrivers(getDriversByCarId(c.getId())));
+        return carList;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String updateCar = "UPDATE cars SET model = ?, manufacturer_id = ? "
+                + "WHERE id = ? AND is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateCarStatement = connection.prepareStatement(updateCar)) {
+            updateCarStatement.setString(1, car.getModel());
+            updateCarStatement.setLong(2, car.getManufacturer().getId());
+            updateCarStatement.setLong(3, car.getId());
+            updateCarStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can`t update car = " + car, e);
+        }
+        deleteRelation(car.getId());
+        createRelation(car);
```

Comment:
your first method accepts id, second method – car instance. Let's make these methods' parameters consistent. And also let's rename them, as their purposes are not clear enough